CHANGELOG.md

```diff
@@ -7,17 +20,6 @@
 
 * First stable release.
 
-## 1.0.0-beta.18
-
-* Add a `CompileRequest.source_map_include_sources` field that tells the
-  compiler to embed the contents of all source files in the generated source
-  maps.
-
-## 1.0.0-beta.17
-
-* Mark `ImportResponse.result` as optional. Importers should be able to return
-  `null` to indicate that a file wasn't found.
-
```

Comment:
I guess these lines are not supposed to be deleted?

Reply:
Yep, merge error. Thanks for catching that!